src/scss/style.scss

```
@@ -11,6 +11,7 @@
 @import 'src/scss/components/menuButton';
 @import 'src/scss/components/rankRecord';
 @import 'src/scss/components/rankSection';
+@import 'components/timer';
```

Jak możesz to ustaw tak, żeby konwencja się zgadzała :D

```suggestion
@import 'src/scss/components/timer';
```

---

done, zapamiętam